src/renderer/Core/Settings/Pages/Appearance/DockSettings.tsx

```diff
@@ -0,0 +1,38 @@
+import { useContextBridge, useSetting } from "@Core/Hooks";
+import { Setting } from "@Core/Settings/Setting";
+import { SettingGroup } from "@Core/Settings/SettingGroup";
+import { Switch } from "@fluentui/react-components";
+import { useTranslation } from "react-i18next";
+
+export const DockSettings = () => {
+    const ns = "settingsAppearance";
+    const { t } = useTranslation();
```

Comment:
You can initialize this already with the right namespace:

```suggestion
    const { t } = useTranslation("settingsAppearance");
```

Then you don't need to specify the namespace everytime when using the `t()` function.